infra/modules/s3/emails.tf

```diff
@@ -0,0 +1,30 @@
+provider "aws" {
```

Comment:
You don't need to have the provider set here, this is handled in provider.tf. 

These resources can be included in s3/main.tf instead of being separated into emails.tf. Otherwise you'll need to reference them from s3/main.tf anyway.

Reply:
Ah ok thanks - that makes sense